### DISCUSSION TITLE
Add showcase section

### DISCUSSION THREAD

**Discussion by LITUATUI** [colinwilson/lotusdocs#103] [Feature Requests]
A showcase section is a nice way to see the theme implemented by different people.

Like these examples:

[Blowfish](https://blowfish.page/users/)
[Hextra](https://imfing.github.io/hextra/showcase/)

What do you think?

**Comment by colinwilson** [comment]
Agreed 👍. Already [working on it](https://github.com/colinwilson/lotusdocs/discussions/27#discussioncomment-6786438). I'll update once this feature is released.
